src/bundle/Resources/views/content/tabs/translations.html.twig

```diff
@@ -15,10 +19,12 @@
     <tbody>
         {% for translation in translations %}
         <tr>
-            <td></td>
+            <td>{{ form_widget(actionsForm.removeTranslations[translation.languageCode], {'attr': {'disabled': not translation.canDelete}}) }}</td>
```

Comment:
the API blocks deletion of the main location and checks user permissions, no need to duplicate before making the call